contracts/twap/CumulativeTwap.sol

```diff
@@ -11,70 +11,72 @@ contract CumulativeTwap is BlockContext {
     //
     // STRUCT
     //
+
     struct Observation {
         uint256 price;
         uint256 priceCumulative;
         uint256 timestamp;
     }
 
-    //
-    // EVENT
-    //
-    event PriceUpdated(uint256 price, uint256 timestamp, uint8 indexAt);
```

Comment:
if we remove this then ChainlinkPriceFeedV2 lost this event, suggest either create CumulativeTwapV2 or add it to ChainlinkPriceFeedV2 & BandPriceFeed ( also suggest to add test, this should be covered and break ChainlinkPriceFeedV2 & BandPriceFeed spec ), or just remove ChainlinkPriceFeedV2 & BandPriceFeed if they're no longer needed

Reply:
Prefer removing both Band & ChainlinkV2!
However, currently don't have time to rewrite tests; is this acceptable? or can we open a ticket to do house-cleaning after submitting commit hash?
PR: https://github.com/perpetual-protocol/perp-oracle-contract/pull/58

Reply:
sure lets remove them afterward if no one is using them

Reply:
ticket: https://app.asana.com/0/1202133750666965/1203615824027497